server/server.go

```diff
@@ -42,6 +43,11 @@ type WysteriaServer struct {
 	middleware_server wcm.EndpointServer
 }
 
+var (
+	reserved_item_facets = []string{"collection"}
+	reserved_ver_facets = []string{"collection", "itemtype", "variant"}
```

Comment:
Just completed major renaming exercise. Thank the Lord for shift + f6 in gogland